apispec/api.spec.json

```diff
@@ -59,6 +59,66 @@
                     }
                 }
             }
+        },
+        "/compliance": {
+            "get": {
+                "tags": [
+                    "services"
+                ],
+                "summary": "verify exports compliance for a given user",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ComplianceScreeningResponse"
+                                },
+                                "examples": {
+                                    "sample1": {
```

Comment:
Sort of a nit, but should we name these `sample*` responses more explicitly? Looking at them when validating the spec in a swagger editor seemed like it might be more readable if they were intention revealing.

Reply:
Also, should we include the potential for a dependency failure response? https://github.com/RedHatInsights/entitlements-api-go/pull/90/files#diff-e7ac0a2dd3ec503a6608c98c6bee18b71671bcae72450a5fa307bbaa55698297R122

Reply:
Yep, good catch. I'll add both of those

- [ ] rename sample responses to reveal intention
- [ ] add dependency failure response to api spec